site_dive_into/settings.py

```diff
@@ -39,6 +38,7 @@
     "django_filters",
     "api",
     "rest_framework",
+    'corsheaders',
```

Comment:
So... do we need a standard for it? I prefer `'` quotes but `"` quotes already exist in our code-base.

Reply:
I fixed it. Thank you